### ISSUE TITLE
SBT version conflict error when `requests` and `os-lib` are used together

### ISSUE THREAD

**Issue by spamegg1** [com-lihaoyi/requests-scala#119]
SBT version is 1.8.0.

Here is `build.sbt`:
```scala
lazy val root = project
  .in(file("."))
  .settings(
    scalaVersion := "3.2.1",
    libraryDependencies ++= Seq(
      "com.lihaoyi"   %% "requests" % "0.7.1",
      "com.lihaoyi"   %% "os-lib"   % "0.9.0",
    )
  )
```
The error:
```
[error] (update) found version conflict(s) in library dependencies; some are suspected to be binary incompatible:
[error] 
[error] 	* com.lihaoyi:geny_3:1.0.0 (early-semver) is selected over 0.6.10
[error] 	    +- com.lihaoyi:os-lib_3:0.9.0                         (depends on 1.0.0)
[error] 	    +- com.lihaoyi:requests_3:0.7.1                       (depends on 0.6.10)
[error] 
[error] 
[error] this can be overridden using libraryDependencySchemes or evictionErrorLevel
[error] Total time: 0 s, completed Dec 15, 2022, 11:02:59 PM
```
Similar errors occur if `scalaVersion := 2.13.10` or `scalaVersion := 2.12.17`.

**Comment by nemoo** [comment]
I have the same problem. It is really annoying.

**Comment by spamegg1** [comment]
I [tried to update the geny version manually](https://github.com/spamegg1/requests-scala/actions/runs/3707677260) but I got a binary incompatibility error.

**Comment by lolgab** [comment]
Released requests `0.8.0` which depends on geny `1.0.0`
